app/frontend/src/locales/fr/translation.json

```diff
@@ -0,0 +1,131 @@
+    "askTitle": "Demandez à vos données",
+    "gpt4vExamples": {
+        "1": "Comparez l'impact des taux d'intérêt et du PIB sur les marchés financiers.",
+        "2": "Quelle est la tendance prévue pour l'indice S&P 500 au cours des cinq prochaines années? Comparez-le aux performances passées de l'S&P 500",
+        "3": "Pouvez-vous identifier une corrélation entre les prix du pétrole et les tendances du marché boursier?",
+        "placeholder": "Exemple: Mon plan couvre-t-il les examens oculaires annuels?"
+    },
+    "generatingAnswer": "génération de la réponse",
+
+    "tooltips": {
+        "submitQuestion": "Soumettre une question",
+        "askWithVoice": "Poser une question à l'aide de la voix",
+        "stopRecording": "Arrêter l'enregistrement de la question",
+        "showThoughtProcess": "Montrer le processus de réflexion",
+        "showSupportingContent": "Montrer le contenu de soutien",
+        "speakAnswer": "Parler réponse",
+        "info": "Info",
+        "save": "Sauvegarder"
+    },
+
+    "headerTexts":{
+        "thoughtProcess": "Processus de Réflexion",
+        "supportingContent": "Contenu d'Appui",
+        "citation": "Citation"
+    },
+
+    "labels": {
+        "headerText": "Configurer la génération de réponses",
+        "closeButton": "Fermer",
+        "promptTemplate": "Modèle de l'invite",
+        "temperature": "Température",
+        "minimumSearchScore": "Score de recherche minimum",
+        "minimumRerankerScore": "Score minimum du reranker",
+        "retrieveCount": "Récupérer ce nombre de résultats de recherche :",
+        "excludeCategory": "Exclure la catégorie",
+        "useSemanticRanker": "Utiliser le classement sémantique pour la récupération",
+        "useSemanticCaptions": "Utiliser les titres sémantiques",
+        "useSuggestFollowupQuestions": "Suggérer des questions de suivi",
+        "useGPT4V": "Utiliser le modèle GPT Vision",
+        "gpt4VInput": {
+            "label": "Entrées du modèle GPT Vision",
+            "options": {
+                "textAndImages": "Images et texte",
+                "images": "Images",
+                "texts": "Texte"
+            }
+        },
+        "retrievalMode": {
+            "label": "Mode de récupération",
+            "options": {
+                "hybrid": "Vecteurs + Texte (Hybride)",
+                "vectors": "Vecteurs",
+                "texts": "Texte"
+            }
+        },
+        "vector": {
+            "label": "Champs de vecteur (recherche de vecteur multi-requête)",
+            "options": {
+                "embedding": "Incorporations de texte",
+                "imageEmbedding": "Incorporations d'images",
+                "both": "Incorporations de texte et d'images"
+            }
+        },
+        "useOidSecurityFilter": "Utiliser le filtre de sécurité oid",
+        "useGroupsSecurityFilter": "Utiliser le filtre de sécurité de groupe",
+        "shouldStream": "Streamer les réponses du chat"
```

Comment:
```suggestion
        "shouldStream": "Diffuser en continu les réponses"
```

Reply:
Thank you, I have made the changes as per your suggestions.